action.js

```diff
@@ -4,8 +4,14 @@ const { isNodeVulnerable } = require('./index')
 async function run () {
   // Inputs
   const nodeVersion = core.getInput('node-version', { required: true })
-  core.info(`Checking Node.js version ${nodeVersion}...`)
-  const isVulnerable = await isNodeVulnerable(nodeVersion)
+  const platform = core.getInput('platform', { required: false })
+
+  if (platform && !['linux', 'win', 'osx'].includes(platform)) {
```

Comment:
`os.platform` is restricted to just these platforms?

Reply:
Yes, based on [the validation](https://github.com/nodejs/security-wg/blob/e3b8f8ae41e11601a45d08238d24d3ecfdfb3502/tools/vuln_valid/vulnValidate.js#L37) done in https://github.com/nodejs/security-wg/pull/912

Reply:
That's why I suggested to use the nodejs/build list instead, we'll be able to support more platforms

Reply:
Extended to match os.platform approach from https://github.com/nodejs/security-wg/pull/912/commits/535ce830904cf1367620372ae3642175a14569a5 in https://github.com/RafaelGSS/is-my-node-vulnerable/pull/12/commits/8e49600a5389c3b518764d06f2b1d60335dd20d2